### ISSUE TITLE
Add the initial "Manage Resources" Side panel workflow

### ISSUE THREAD

**Issue by marcellamaki** [learningequality/kolibri#12786]
## Overview

The lesson management workflow is being aligned to use the similar side panel management UX that we introduced in quizzes in 0.17. As we work on extending some of the technical aspects of that work (managing the selection of resources and folders, tracking and updating local state, etc.), we will work in parallel to build out the components and pages that surround those parts of the workflow. 

This issue is to update the initial page when a user opens the first manage lesson side panel. The new temporary routes, added in https://github.com/learningequality/kolibri/pull/12672 should be used. 

![Image](https://github.com/user-attachments/assets/06120e55-1178-48da-9463-5e5d953f4ff0)

### Acceptance Criteria

The cards used for this work will be built on top of the new `KCard`, and they require the use of the new `KCardGrid`. Please review the latest design system documentation about the new card and grid. Remember that the "deployed" version of KDS documentation on netlify is v4, which does not contain the new cards. Please use the latest [version 5 documentation](https://deploy-preview-796--kolibri-design-system.netlify.app/kcard/) as guidance.

Page layout 
- [ ] Create the first "landing page" for manage lesson resources at the temporary path
- [ ] The side panel header should be an H1, the subheaders should be h2
- [ ] Add the new "search" button (right now, this does not need to do anything, but it should be added so the page layout is correct)
- [ ] the X on the top of the side panel should close the panel, as should the "save and finish" and the "ESC" key. (For now, no validations are required)

Bookmarks 
- [ ] The card should be built with the new `KCard` and match the spec design  
- [ ] The card should essentially function as a link, which should open the bookmark side panel
- [ ] The bookmarks do not need to display on bookmark side panel open, as this depends on the selection state management WIP

Channel cards
- [ ] The channel cards should be rendered using the [updated Channel Card](https://github.com/learningequality/kolibri/pull/12607), which is build on top of the new `KCard`. This version was created as an MVP and may need some slight modification to match the spec. Please discuss any questions or decisions with @marcellamaki 

### Out of scope
- [ ] Updating the cards to add and display the metadata is out of scope for this issue. The channel cards do not have metadata tags yet, and this requires some backend prerequisite work. 
- [ ] Any changes to the pagination of channel display - for now we will use the same scroll behavior as is currently used on the channel 
- [ ] Adding the "shopping cart" link on the bottom bar
- [ ] "Connecting" the lesson path to the "manage lesson resources" button. This stays in a parallel track of work. Please make sure to add this as reviewer guidance and put the path that QA testers should use in the URL of your PR.

## Resources
- [Figma](https://www.figma.com/design/lVJt5ukOrxS9qay0rXy7GC/0.18-Coach-updates?node-id=107-20613&node-type=frame&t=BtL8UCkGoj5bhl9p-0) 
- [KDS docs](https://deploy-preview-796--kolibri-design-system.netlify.app/kcard/)
